packages/simulation-service/tsconfig.src.json

```diff
@@ -5,7 +5,8 @@
   },
   "include": [
     "src",
-    "simulation-service.ts"
-  ],
+    "simulation-service.ts", 
+    "src/direction.ts", 
```

Comment:
direction and isimulation is added automatically by line 7 "src". You can remove these two lines :)